src/SWOTRiver/SWOTRiverEstimator.py

```diff
@@ -1173,78 +1150,84 @@ def compute_enhanced_slopes(
         node height using Gaussian moving average.
         For more information, pleasec see Dr. Renato Frasson's paper:
         https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2017WR020887
-
         inputs:
         river_reach_collection: collection of river_reach instance
         max_window_size: the max of Gaussian window, default is 10km
         min_sigma : min sigma for gaussian averaging, default is 1km
         window_size_sigma_ratio : default is 5
-
         output:
         enhanced_slopes: enhanced reach slopes
         """
         # get list of reach index
         n_reach = len(river_reach_collection)
-        ind = [reach.reach_indx[0] for reach in river_reach_collection]
-
+        
+        ind=[]
```

Comment:
I'm not convinced you need to do this, won't this cause problems later in the reach_downstream / reach_upstream?  if the list reach_downstream.s is empty it won't cause issue at the later compuation.  I'd want to test before merging